core/templates/dev/head/base.html

```diff
@@ -280,6 +281,16 @@
               {{ warnings_and_loader() }}
             </div>
 
+            <noscript>
+              <div class="oppia-page-cards-container">
+                <div class="md-default-theme oppia-page-card oppia-long-text">
+                  <h2>We Need JavaScript in Your Browser <img src="/images/general/sad_face.png" /> </h2>
```

Comment:
I used "we" in the error message because it removes any blame from the user.  

As for the icon, that is the icon I used; I just changed the filename.  I didn't see any material design icons, so I created a directory "material design" under 'static/images' and put the icon in there.  That way we have a place to put other material design icons or images.